modules/golang/golang

```diff
@@ -1,5 +1,11 @@
 compdef _golang go
 
+function _deps(){
+	app="go"
+
+	$(hash $app 2>/dev/null) || $(myzsh error "Couldn't find application ${app}" && return 1)
```

Comment:
Should $() not be used here? I originally had it using only () and that didn't seem to bode well.